packages/desktop/components/AccountManagementDetails.svelte

```diff
@@ -123,126 +123,130 @@
-                    >
-                    <CopyableBox
-                        clearBackground
-                        clearBoxPadding
-                        isCopyable
-                        value={address}
-                        classes="flex space-x-2 items-center"
-                    >
+                {#if isImplicitAccount}
+                    <div class="flex flex-col space-y-2">
+                        <Text color="gray-600" fontWeight={FontWeight.medium} fontSize="12" type={TextType.p}
+                            >{localize('views.accountManagement.details.mana')}</Text
+                        >
                         <Text type={TextType.pre} fontSize="13" lineHeight="leading-120" classes="text-start"
-                            >{address}</Text
+                            >{selectedOutput.output?.mana}</Text
                         >
-                        <Icon icon={IconEnum.Copy} classes="text-blue-500" width={24} height={24} />
-                    </CopyableBox>
-                </div>
-            {/if}
-            {#if isImplicitAccount}
-                <div class="flex flex-col space-y-2">
-                    <Text color="gray-600" fontWeight={FontWeight.medium} fontSize="12" type={TextType.p}
-                        >{localize('views.accountManagement.details.mana')}</Text
-                    >
-                    <Text type={TextType.pre} fontSize="13" lineHeight="leading-120" classes="text-start"
-                        >{selectedOutput.output?.mana}</Text
-                    >
-                </div>
-            {/if}
-            {#if accountId && primaryKey}
-                <div class="flex flex-col space-y-2">
-                    <Text color="gray-600" fontWeight={FontWeight.medium} fontSize="12" type={TextType.p}
-                        >{localize('views.accountManagement.details.key')}</Text
-                    >
-                    <Text type={TextType.pre} fontSize="13" lineHeight="leading-120" classes="text-start"
-                        >{primaryKey}</Text
-                    >
-                </div>
+                    </div>
+                {/if}
+                {#if accountId && primaryKey}
+                    <div class="flex flex-col space-y-2">
+                        <Text color="gray-600" fontWeight={FontWeight.medium} fontSize="12" type={TextType.p}
+                            >{localize('views.accountManagement.details.key')}</Text
+                        >
+                        <Text type={TextType.pre} fontSize="13" lineHeight="leading-120" classes="text-start"
+                            >{primaryKey}</Text
+                        >
+                    </div>
+                {/if}
+            </right-pane-wrapper>
+            {#if isMainAccount && hasMainAccountNegativeBIC}
+                <TextHint variant={TextHintVariant.Danger} text={localize('views.accountManagement.details.hint')} />
```

Comment:
I would bve a bit more specific about this locale, an account detail hint is a bit genric